src/NexusMods.App.UI/Controls/Trees/Files/IFileTreeNodeViewModel.cs

```diff
@@ -54,17 +54,36 @@ string ToFormattedFileCount()
         return FileCount > 0 ? FileCount.ToString() : string.Empty;
     }
 
-    string ToFormattedChangeState()
+    string FormattedChangeState
     {
-        return ChangeType switch
+        get
         {
-            FileChangeType.Added => Language.IFileTreeNodeViewModel_ToFormattedChangeState_Added,
-            FileChangeType.Modified => IsFile
-                ? Language.IFileTreeNodeViewModel_ToFormattedChangeState_Modified
-                : Language.IFileTreeNodeViewModel_ToFormattedChangeState_Contents_modified,
-            FileChangeType.Removed => Language.IFileTreeNodeViewModel_ToFormattedChangeState_Removed,
-            FileChangeType.None => string.Empty,
-        };
+            return ChangeType switch
+            {
+                FileChangeType.Added => Language.IFileTreeNodeViewModel_ToFormattedChangeState_Added,
+                FileChangeType.Modified => IsFile
+                    ? Language.IFileTreeNodeViewModel_ToFormattedChangeState_Modified
+                    : Language.IFileTreeNodeViewModel_ToFormattedChangeState_Contents_modified,
+                FileChangeType.Removed => Language.IFileTreeNodeViewModel_ToFormattedChangeState_Removed,
+                FileChangeType.None => string.Empty,
+            };
+        }
+    }
+    
+    string FormattedChangeStateToolTip
+    {
+        get
+        {
+            return ChangeType switch
+            {
+                FileChangeType.Added => Language.IFileTreeNodeViewModel_FormattedChangeStateToolTip_Added,
+                FileChangeType.Modified => IsFile
+                    ? Language.IFileTreeNodeViewModel_FormattedChangeStateToolTip_ModifiedFile
+                    : Language.IFileTreeNodeViewModel_FormattedChangeStateToolTip_ModifiedFolder,
+                FileChangeType.Removed => Language.IFileTreeNodeViewModel_FormattedChangeStateToolTip_Removed,
+                FileChangeType.None => string.Empty,
+            };
+        }
     }
```

Comment:
Please make two methods for this. Huge `get` blocks shouldn't exist.

Reply:
I still need the properties to bind to from XAML, but refactored away the get contents into methods